src/app/core/entity-components/entity-details/form/form.component.scss

```
@@ -46,3 +46,8 @@
   padding-left: 10px;
   color: black;
 }
+
+.mat-icon{
+  font-size: 1em;
```

```suggestion
  font-size: 1.25em;
  color: grey;
```
let's make them a bit bigger to make it an easier target especially on mobile